samples/README.md

```diff
@@ -75,11 +75,11 @@ For better readability and integrations with the sample test infrastructure, sam
 * The sample file should be either `*.py` or `*.ipynb`, and its file name is consistent with its directory name.
 * For `*.py` sample, it's recommended to have a main invoking `kfp.compiler.Compiler().compile()` to compile the 
 pipeline function into pipeline yaml spec.
-* For `*.ipynb` sample, parameters (e.g., `experiment_name` and `project_name`)
+* For `*.ipynb` sample, parameters (e.g., `project_name`)
```

Comment:
> Shall we explicitly forbidden setting the experiment name in notebook samples?

The sample test uses the `OVERRIDE` env var, so it will still work (with a warning).

>shall we direct the user not to install kfp released version here as well?

Using `--upgrade` should probably be prohibited.
Using version constraint like `python3 -m pip install 'kfp>=0.1.31'` should be fine (the master version should always be at leas as high as the PyPI version). We need to verify that though.

Reply:
I see. So `OVERRIDE` experiment name has the highest priority.